cms/migrations/0037_learning_outcomes_subhead_richtext.py

```diff
@@ -1,39 +1,41 @@
 # Generated by Django 2.1.9 on 2019-07-03 09:59
-import re
 import wagtail.core.fields
+from bs4 import BeautifulSoup
 from django.db import migrations
-from django.db.models import F, Value
-from django.db.models.functions import Concat
 
 
-def wrap_in_paragraph(apps, schema_editor):
+def dummy_method(apps, schema_editor):  # pylint: disable=unused-argument
```

Comment:
You can replace this with a library method. Look [here](https://github.com/mitodl/mitxpro/blob/master/courses/migrations/0016_enrollment_order.py#L115) for an example

Reply:
Replaced with `RunPython.noop`.